### ISSUE TITLE
triaining the resnet50 model

### ISSUE THREAD

**Issue by xiaoshudi** [deepinsight/insightface#1355]
Hi, i used the vggface2 to train the resnet50 model, the config is in blow.
batch size:512
metric:arcface
scale:64
margin:0.5
start learning rate:0.1
The accuracy of the training data vggface2 is 0 all the time. the loss goes down normally. The verification accuracy in LFW  is normally increased.

Could you tell me what happened when you were training and why?
